srcpkgs/android-udev-rules/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'android-udev-rules'
 pkgname=android-udev-rules
-version=20230104
+version=20230303
 revision=1
 short_desc="Android udev rules list aimed to be the most comprehensive on the net"
-maintainer="Rien Maertens <rien.maertens@posteo.be>"
+maintainer="Mohammed Anas <triallax@tutanota.com>"
```

Comment:
@rien can they adopt this from you?

Reply:
Yes, I'm no longer using void linux. So go ahead.